packages/eslint-plugin/src/rules/no-misused-promises.ts

```diff
@@ -22,6 +30,14 @@ export default util.createRule<Options, 'conditional' | 'voidReturn'>({
     messages: {
       voidReturn:
```

Comment:
Could you rename this to `voidReturnArgument`? Having it be the only one without a third word makes it seem like it's a more general complaint than the others.